Source/WebCore/platform/graphics/cv/VideoFrameCV.mm

```diff
@@ -168,11 +178,21 @@ static vImage_Buffer makeVImageBuffer8888(CVPixelBufferRef buffer)
 
 RefPtr<VideoFrame> VideoFrame::createBGRA(std::span<const uint8_t> span, size_t width, size_t height, const ComputedPlaneLayout& plane, PlatformVideoColorSpace&& colorSpace)
 {
+    void *pixelBufferData = nullptr;
+    if (!::WTF::tryFastMalloc(width * height * 4).getValue(pixelBufferData))
+        return nullptr;
+
+    auto releasePixelBufferData = [] (void* context, const void*) {
+        ::WTF::fastFree(context);
+    };
+
     CVPixelBufferRef rawPixelBuffer = nullptr;
+    auto status = CVPixelBufferCreateWithBytes(kCFAllocatorDefault, width, height, kCVPixelFormatType_32BGRA, pixelBufferData, width * 4, releasePixelBufferData, pixelBufferData, nullptr, &rawPixelBuffer);
 
-    auto status = CVPixelBufferCreate(kCFAllocatorDefault, width, height, kCVPixelFormatType_32BGRA, nullptr, &rawPixelBuffer);
-    if (status != noErr || !rawPixelBuffer)
+    if (status != noErr || !rawPixelBuffer) {
+        ::WTF::fastFree(pixelBufferData);
```

Comment:
I'd like to avoid this, since it's too easy for someone changing the code (or adding a new version, e.g. for a different color channel type) to miss that the releasePixelBufferData call wouldn't happen here.

It should be possible to pull this into the wrapper as well though